packages/eventual-send/src/index.js

```diff
@@ -109,34 +182,36 @@ export function makeHandledPromise() {
         if (resolved) {
           return resolvedTarget;
```

Comment:
`handleResolve`'s return value is never used. Should it be void return instead?

Reply:
I'll remove the return value from it, but it isn't actually typed.  I ran into trouble when trying to type it because the Typescript typings of `WeakMap` are too strict.  They say `WeakMap.prototype.has(key: K)` instead of `key: any`.

There's still further typing that can be done, but I'm probably leaving that until after `@agoric/eventual-send` is adopted into `@endo` and aligned with the latest proposal.